cmd/instances.go

```diff
@@ -653,6 +654,49 @@ var oauthClientInstanceCmd = &cobra.Command{
 	},
 }
 
+var findOauthClientCmd = &cobra.Command{
+	Use:   "find-oauth-client <domain> <software_id>",
+	Short: "Finds a OAuth client",
+	Long:  `Search an oauth client from his software_id`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+		c := newAdminClient()
+
+		body := struct {
+			Domain     string `json:"domain"`
+			SoftwareID string `json:"software_id"`
+		}{
+			Domain:     args[0],
+			SoftwareID: args[1],
+		}
+		// Make request
+		b, err := json.Marshal(body)
+		req := &request.Options{
+			Method: "GET",
+			Path:   "instances/oauth_client",
+			Body:   bytes.NewReader(b),
```

Comment:
HTTP specs say that GET with a body has no semantics, ie it shouldn't be used. We can use a query-string for these 2 parameters instead.